specs/capabilities.md

```diff
@@ -1,7 +1,7 @@
+| Multi-Activity Agreement      | 0.5.0                 | Yes                   | Negotiate ability to create multiple activities under single Agreement. Use (which?) property in Demand/Offer to indicate node's support for Multi-Activity. If counterparty does not support Multi-Activity, the node falls back to single Activity per Agreement behaviour.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| Restart Proposal Negotiations | 0.7.0                 | Yes                   | Agent is allowed to restart negotiations, by sending `Counter Proposal`, after he rejected Proposal at some point. Counter-party will receive regular `ProposalEvent` in this case. Only Agent rejecting Proposal has initiative in restarting negotiations, rejected Agent can only wait for this to happen. To indicate, that Proposal rejection isn't final and negotiations can be restarted later, Agent can set `golem.proposal.rejection.is-final` (bool) field in `Reason` structure. If this value is set to false, Agent can free any state related to this negotiation. The same field can be set in `Reason` sent in `Reject Agreement` operation. Requestor can send new counter Proposal after some period of time or propose the same Agreement for the second time. (No change to specification) |
+| Payment version protocol      | 0.15.0                | No                    | Specify what payment protocol version is implemented in yagna client. It enables adding new features to yagna like payments via contract and checking if other                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Sentence is not finished.
And please add link to the description in `payment_version.md`